src/commands/weather.ts

```diff
@@ -48,5 +48,5 @@ export default <Command>{
 }
 
 function timeConverter(UNIXTimestamp: number) {
-    return new Date(UNIXTimestamp * 1000).toLocaleString('fr-FR')
+    return new Date(UNIXTimestamp * 1000 + 7200).toLocaleString('fr-FR')

```

Comment:
Le format devient HH:mm, ca allège, et on a pas besoin de plus d'info

```suggestion
    return new Date(UNIXTimestamp).toTimeString().slice(0, 5);
```